hono/src/services/items.ts

```diff
@@ -65,20 +65,19 @@ app.put(
   async (c) => {
     const params = c.req.valid('param')
     const body = c.req.valid('json')
-
-    return c.json(
-      (
-        await c
-          .get('db')
-          .update(items)
-          .set({
-            ...body,
-            authorUsername: 'rak3rman', // TODO: remove and replace with author integration
-          })
-          .where(eq(items.id, params.id))
-          .returning()
-      )[0]
-    )
+    const res = (
+      await c
+        .get('db')
+        .update(items)
+        .set({
+          ...body,
+          authorUsername: 'rak3rman', // TODO: remove and replace with author integration
```

Comment:
Can this just be an ENV var?

Reply:
This is hardcoded for now awaiting implementation in issue #48